### PR TITLE
Fix fastpitch test nightly

### PR DESCRIPTION
**Pull request by hsiehjackson**
# What does this PR do ?

Fix multi-speaker FastPitch test nightly

**Collection**: [TTS]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [x] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [x] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: tests/collections/tts/models/test_fastpitch.py [hunk 1]
```diff
@@ -38,4 +41,23 @@ def test_inference(pretrained_model, language_specific_text_example):
...
     parsed_text = model.parse(text)
-    _ = model.generate_spectrogram(tokens=parsed_text)
+
+    # Multi-Speaker
+    speaker_id = None
+    reference_spec = None
+    reference_spec_lens = None
+
+    if hasattr(model.fastpitch, 'speaker_emb'):
+        speaker_id = 0
```
Comment (rlangman) [inline tests/collections/tts/models/test_fastpitch.py:51]: Nitpick: could create a helper function to handle the branching logic easier, like:

```
def _get_inputs(model):
    if hasattr(model.fastpitch, 'speaker_emb'):
        return 0, None, None
    if not hasattr(model.fastpitch, 'speaker_encoder'):
        None, None, None
    ...

```

**Review by rlangman — APPROVED** [review]

File: tests/collections/tts/models/test_fastpitch.py (hunk 1 above)
Comment (XuesongYang) [reply tests/collections/tts/models/test_fastpitch.py:51]: agreed. but it seems this add-ons are only applied to fastpitch models, not all. It looks good to just keep as-is.

**Review by XuesongYang — APPROVED** [review]
LGTM. Thanks for your quick fix.

**Comment by XuesongYang** [comment]
could you please also file the same PR to r1.19.0 branch?

**Review by XuesongYang — APPROVED** [review]

**Pull request merged**